### ISSUE TITLE
Formula of batchnorm

### ISSUE THREAD

**Issue by unipdjahanifar** [sovrasov/flops-counter.pytorch#99]
Hello
Could you please let me know the formula that is behind the calculation of number of macs for batch normalization?

**Comment by sovrasov** [comment]
Hi. See https://github.com/sovrasov/flops-counter.pytorch/blob/master/ptflops/pytorch_ops.py#L44

**Comment by unipdjahanifar** [comment]
Thanks. Could you please let me what are these?

module, input and output

**Comment by unipdjahanifar** [comment]
Thanks a lot. based on the shred codes, could you please let me know whether this statement is correct: 
"in batch normalization layer we multiply the input with 2 to compute the number of macc?"

**Comment by unipdjahanifar** [comment]
Thanks a lot. Could you please let me know whether is there any formula for the backward? your kind support in this regard will be appreciated.

**Comment by sovrasov** [comment]
All the formulas related to BN can be fount in the original paper: https://arxiv.org/pdf/1502.03167.pdf
